app/views/users/embedded.html.erb

```diff
@@ -20,7 +20,7 @@
           <div class="slidecard-description"><%= strip_tags(slide.description).truncate(80) %></div>
           </div>
           <div class="slidecard-information">
-            <div class="h5" style="margin-top:2px;margin-bottom:2px;">by <a href="<%= "#{root_url}users/#{slide.user_id}" %>" target="_blank"><%= strip_tags slide.user_display_name %></a></div>
+            <div class="h5" style="margin-top:2px;margin-bottom:2px;">by <a href="<%= "#{root_url}#{slide.user.username}" %>" target="_blank"><%= strip_tags slide.user_display_name %></a></div>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)